hudi-common/src/main/java/org/apache/hudi/metadata/HoodieBackedTableMetadata.java

```diff
@@ -537,7 +538,8 @@ public HoodieTableMetaClient getMetadataMetaClient() {
   }
 
   public Map<String, String> stats() {
-    return metrics.map(m -> m.getStats(true, metadataMetaClient, this)).orElse(new HashMap<>());
+    Set<String> allMetadataPartitionPaths = Arrays.stream(MetadataPartitionType.values()).map(MetadataPartitionType::getPartitionPath).collect(Collectors.toSet());
+    return metrics.map(m -> m.getStats(true, metadataMetaClient, this, allMetadataPartitionPaths)).orElse(new HashMap<>());
```

Comment:
Do we need to fetch the enabled partitions here instead of all?

Reply:
HoodieMetadataMetrics.getStats(boolean detailed, HoodieTableMetaClient metaClient, HoodieTableMetadata metadata) 

reloads the timeline. 
can we move the reload to outside of the caller so that we don't reload for every MDT partition stats

Reply:
Removed the reload of timeline. It is actually not required since the code is called right after commit where the metaClient is reloaded anyways.